src/adapter.js

```diff
@@ -191,6 +191,18 @@ function KarmaReporter (tc, jasmineEnv) {
 
   this.jasmineDone = function (result) {
     result = result || {}
+
+    // Any errors in top-level afterAll blocks are given here.
+    if (result.failedExpectations && result.failedExpectations.length) {
```

Comment:
Thanks

Only one suggest - extract the code to a function and remove duplication between **jasmineDone** and **suiteDone***